### ISSUE TITLE
Carousel versioning issues with Flutter upgrade

### ISSUE THREAD

**Issue by Teejayblaze** [gbrvalerio/carousel#13]
After upgrading my flutter version, i notice that my carousel code breaks.
Here is the error generated:
The current Dart SDK version is 2.1.0-dev.5.0.flutter-a2eb050044.

Because package depends on carousel any which requires SDK version <2.0.0, version solving failed.

pub get failed (1)

**Comment by sjmcdowall** [comment]
THat's correct alas -- someone (?) needs to update the package to be Dart 2+ compliant.  Is the author even around? @gbrvalerio ??
